era5cli/fetch.py

```diff
@@ -59,17 +59,20 @@ class Fetch:
             be written to stdout.
         prelimbe: bool
             Whether to download the preliminary back extension (1950-1978).
+        land: bool
+            Whether to download ERA5-Land data.
     """
 
     def __init__(self, years: list, months: list, days: list,
                  hours: list, variables: list, outputformat: str,
                  outputprefix: str, period: str, ensemble: bool,
                  statistics=None, synoptic=None, pressurelevels=None,
-                 merge=False, threads=None, prelimbe=False):
+                 merge=False, threads=None, prelimbe=False, land=False):
         """Initialization of Fetch class."""
         self.months = era5cli.utils._zpad_months(months)
         """list(str): List of zero-padded strings of months
         (e.g. ['01', '02',..., '12'])."""
+        # TODO make sure these docstrings are consistent with cli docs
```

Comment:
Does this still need doing? :-)

Reply:
Yes, but it is probably better done in a separate pull request.

Reply:
Addressing the docstring consistency may also fit addressing issue #68, and ensuring input is verified comprehensively. It is beyond the scope of this PR.

Reply:
Added an issue #71 and removing the #TODO :)